libbeat/outputs/elasticsearch/client.go

```diff
@@ -50,14 +50,14 @@ type Connection struct {
 
 // Metrics that can retrieved through the expvar web interface.
 var (
-	ackedEvents            = expvar.NewInt("libbeatEsPublishedAndAckedEvents")
-	eventsNotAcked         = expvar.NewInt("libbeatEsPublishedButNotAckedEvents")
-	publishEventsCallCount = expvar.NewInt("libbeatEsPublishEventsCallCount")
-
-	statReadBytes   = expvar.NewInt("libbeatEsPublishReadBytes")
-	statWriteBytes  = expvar.NewInt("libbeatEsPublishWriteBytes")
-	statReadErrors  = expvar.NewInt("libbeatEsPublishReadErrors")
-	statWriteErrors = expvar.NewInt("libbeatEsPublishWriteErrors")
+	ackedEvents            = expvar.NewInt("libbeat.es.published_and_acked_events")
+	eventsNotAcked         = expvar.NewInt("libbeat.es.published_but_not_acked_events")
+	publishEventsCallCount = expvar.NewInt("libbeat.es.call_count.PublishEvents")
```

Comment:
I left it by that because it's the name of the function. So how many times the `PublishEvents` function was called. So it felt more right to break the rules here.